ignite/pkg/cosmosgen/templates/rest/rest.ts.tpl

```diff
@@ -1,8 +1,6 @@
-// FOR REFERENCE ONLY -- NOT A TEMPLATE
-
 import axios, { AxiosInstance, AxiosRequestConfig, AxiosResponse, ResponseType } from "axios";
-import { QueryAllBalancesResponse, QueryBalanceResponse, QueryDenomMetadataByQueryStringResponse, QueryDenomMetadataResponse, QueryDenomOwnersByQueryResponse, QueryDenomOwnersResponse, QueryDenomsMetadataResponse, QueryParamsResponse, QuerySpendableBalancesResponse, QuerySupplyOfResponse, QueryTotalSupplyResponse } from "./module";
-
+{{ range .HTTPQueries }}import { Query{{ .Name }}Response } from "{{ resolveFile .FilePath }}";
```

Comment:
Unfortunately not all queries follow this pattern:

e.g.  Note lack of `Query` here:

https://github.com/cosmos/cosmos-sdk/blob/main/proto/cosmos/auth/v1beta1/query.proto#L59-L74

Reply:
fixed https://github.com/ignite/cli/pull/4706/commits/42eac760a96dfc72a956da6727b1f32bd0b7ba11